src/main/generic/Core.js

```diff
@@ -16,7 +16,11 @@ class Core {
         this.consensus = new Consensus(this.blockchain, this.mempool, this.network);
 
         // Wallet
-        this.wallet = await Wallet.getPersistent();
+        if (walletSeed === undefined) {
```

Comment:
Be careful! I'm not sure whether this works as supposed to.
[Here](https://github.com/nimiq-network/core/pull/168/files#diff-0c66691a731f5ff57c3398eaffbe3aa8R16) (`clients/nodejs/index.js`, line 16), you initialise `walletSeed` as `null` if not specified. Now you're checking against `undefined`.